src/web/pubsub-form-table.jsp

```diff
@@ -0,0 +1,151 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+
+<!--
+Parameters:
+fields - fields from a data form (needs to be set as request scope)
+nonDisplayFields - a list of field names that shouldn't be displayed (needs to be set as request scope)
+listTypes - a map of field.variables to listType (user or group)
+errors - a map of field.variable to error strings
+detailPreFix - property prefix for additional detail to be displayed against the fields on the form
+ -->
+
+<script>
+function clearSelected(id){
+    var elements = document.getElementById(id).options;
+
+    for(var i = 0; i < elements.length; i++){
+      elements[i].selected = false;
+    }
+  }
+
+  function delteTableRow(rowId) {
```

Comment:
At least you consistently mistype 'delete' :)

Reply:
Done.